src/cli/advanced/near.rs

```diff
@@ -145,7 +145,7 @@ pub enum WriteCommand {
         #[clap(short, long)]
         wasm_path: String,
         /// Unique identifier for the chain. The default value is 1313161556 (Aurora localnet).
-        /// See https://chainlist.org/ for a list of taken chain IDs.
+        /// See `https://chainlist.org/` for a list of taken chain IDs.
```

Comment:
`[Chainlist](https://chainlist.org/)`
Using markdown-style links might be more readable in documentation preview tools.